action/completion.go

```diff
@@ -51,3 +54,47 @@ source <(gopass completion bash)
 
 	return nil
 }
+
+// CompletionDMenu returns a script that starts dmenu
+// Usage: eval "$(gopass completion dmenu)"
+func (s *Action) CompletionDMenu(c *cli.Context) error {
+	typeit := c.Bool("type")
+
+	list, err := s.Store.List()
+	if err != nil {
+		return err
+	}
+
+	name, err := dmenu(list)
+	if err != nil {
+		return err
+	}
+
+	content, err := s.Store.First(name)
+	if err != nil {
+		return err
+	}
+
+	if typeit {
+		return exec.Command("xdotool", "type", "--clearmodifiers", string(content)).Run()
```

Comment:
So `xdotool` is a _fake keyboard/mouse input, window management, and more_. It is used to type the secret into the focused element on your screen. This can be a terminal, Slack Desktop App. Everything that has a keyboard input. It types the secret's chars into that field one after another.